sdk/purview/azure-purview-workflow/azure/purview/workflow/aio/_operations/_operations.py

```diff
@@ -20,8 +20,7 @@
     map_error,
 )
 from azure.core.pipeline import PipelineResponse
-from azure.core.pipeline.transport import AsyncHttpResponse
-from azure.core.rest import HttpRequest
+from azure.core.rest import AsyncHttpResponse, HttpRequest
```

Comment:
same for `aio/_operations`: https://github.com/Azure/azure-sdk-for-python/pull/32516/files#r1372587895

Reply:
Removed, thanks.